deploy/charts/external-secrets/values.yaml

```
@@ -55,17 +55,17 @@ image:
 installCRDs: true
 
 crds:
-  # -- If true, create CRDs for Cluster External Secret.
+  # -- If true, create CRDs for Cluster External Secret. If set to false also set processClusterExternalSecret: false.
```

It is unclear if this was going to automagically set that other flag to false or if it is a user responsibility. From the OG issue I take it is users' responsibility.

```suggestion
  # -- If true, create CRDs for Cluster External Secret. If set to false you must also set processClusterExternalSecret: false.
```

---

Hm, I think the wording here is pretty clear `if set to false also set` this means SET you set it. It's quite clear to me. :D But I don't mind adding extra clarity.